test/python/transpiler/test_unroller.py

```diff
@@ -31,22 +34,6 @@ def test_basic_unroll(self):
         self.assertEqual(len(op_nodes), 1)
         self.assertEqual(op_nodes[0][1]["op"].name, 'u2')
 
-    def test_unroll_no_basis(self):
```

Comment:
let's restore this test, and check that it raises an error now, because the basis to unroll to is not explicitly given.